modules/security-compliance-nist.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-compliance.adoc
+
+[id="security-compliance-nist_{context}"]
+= Understanding compliance and risk management
+
+FIPS compliance is one of the most critical components required in
+highly secure environments, to ensure that only supported cryptographic
+technologies are allowed on nodes.
+In essence, FIPS mode can be
+enabled by adding `fips: true` to your `install-config.yaml` file during
```

Comment:
This is not accurate. "adding `fips: true` to your `install-config.yaml` file" will only work for the Red Hat Enterprise Linux CoreOS (RHCOS) machines. 
If you add RHEL nodes to your cluster, ensure that you enable FIPS mode on the machines before their initial boot. Details steps seen from https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations#sec-Enabling-FIPS-Mode
You could also refer to https://docs.openshift.com/container-platform/4.3/installing/installing-fips.html for more details.

Reply:
For line 11~13, can we put a link to https://docs.openshift.com/container-platform/4.5/installing/installing-fips.html#installing-fips-mode_installing-fips instead? The content in line 11~13 is kind of duplicacte with the content in the link.

Reply:
I removed the text about how to enable FIPS and added an "Additional resources" section just after this module.